### DISCUSSION TITLE
Using older version of Flyway

### DISCUSSION THREAD

**Discussion by luca-bassoricci** [quarkusio/quarkus#25811] [Q&A] [answered]
I'm using latest Quarkus which includes Flyway v8.5.9.
My SQLServer instance (2016) is older than 5 years so Flyway is not supporting it.
As temporary workaround I pinned Flyway to version 7.15.0 (the latest version with older-than-five-years RDMBS support) in pom.xml, but I'm not sure this is the right way especially if I want to upgrade Quarkus to future versions.
Which is the right way to proceed?

Any advice?
Luca

**Comment by maxandersen** [comment]
That is the right way to attempt it. Test it and if it works you know. Once you upgrade quarkus test again. Quarkus can't guarantee it will work as we are not testing quarkus with all combinations of old dependencies.

[a comment by an automated account is not shown]
